lib/resources/submissions.js

```diff
@@ -82,7 +101,7 @@ module.exports = (service, endpoint) => {
     };
 
     // Nonstandard REST; OpenRosa-specific API.
-    service.post(path, multipart.any(), multerUtf, endpoint.openRosa(({ Forms, Submissions, SubmissionAttachments }, { params, files, auth, query, userAgent }) =>
+    service.post(path, multipart.any(), multerUtf, multipartErrorHandler, endpoint.openRosa(({ Forms, Submissions, SubmissionAttachments }, { params, files, auth, query, userAgent }) =>
```

Comment:
Could you say a little about why this is the only endpoint that needs this error handler? Is it because this endpoint is the only one that uses multer?

Reply:
Yes.  A few of suggestions to improve clarity:

1. colocate `multipart` and `multipartErrorHandler` in this file
2. colocate `multipart` and `multipartErrorHandler` in e.g. `utils/multipart.js`
3. add a comment above `multipart` like `NOTE always use in conjunction with multipartErrorHandler`

Would any of these help?

Reply:
I've gone with #1 for now.